code/obj/item/kendo.dm

```diff
@@ -0,0 +1,250 @@
+//=====
+//Armor
+//=====
+
+/obj/item/clothing/head/helmet/men
+    name = "men"
+    desc = "面 : A light padded helmet with a grilled faceplate to protect the user in a kendo match."
+    icon_state = "men"
+    item_state = "men"
+    seal_hair = 1
+
+    setupProperties()
+        ..()
+        setProperty("coldprot", 10)
+        setProperty("heatprot", 5)
+        setProperty("meleeprot_head", 2)
+
+/obj/item/clothing/suit/armor/douandtare
+    name = "dou and tare"
+    desc = "胴と垂れ : A breastplate and padded skirt used primarily in kendo."
+    icon_state = "dou-tare"
+    item_state = "dou-tare"
+    body_parts_covered = TORSO | LEGS
+    c_flags = ONESIZEFITSALL
+    bloodoverlayimage = SUITBLOOD_ARMOR
+
+    setupProperties()
+        ..()
+        setProperty("coldprot", 10)
+        setProperty("meleeprot", 4)
+        setProperty("rangedprot", 0.4)
+        setProperty("pierceprot", 1)
+        setProperty("movespeed", 1)
+
+/obj/item/clothing/gloves/kote
+    name = "kote"
+    desc = "小手 : Big poofy gloves to cover the hands in kendo sparring."
+    icon_state = "kote"
+    item_state = "kote"
+    material_prints = "navy blue, synthetic leather fibers"
+    crit_override = 1
+    bonus_crit_chance = 0
+    stamina_dmg_mult = 0.35
+
+    setupProperties()
+        ..()
+        setProperty("coldprot", 7)
+        setProperty("conductivity", 0.3)
+
+//======
+//Shinai
+//======
+
+/obj/item/shinai
+    name = "shinai"
+    desc = "竹刀 : A sword-like weapon made of slats of bamboo. Shinai are made to reflect the weight of a katana, but disperse impact on hit to minimize damage."
+    icon = 'icons/obj/items/weapons.dmi'
+    inhand_image_icon = 'icons/mob/inhand/hand_weapons.dmi'
+    icon_state = "shinai"
+    item_state = "shinai-light"
+
+    w_class = 4.0
+    two_handed = 1
+    throwforce = 4.0
+    throw_range = 4
+    stamina_crit_chance = 2
+
+    //these combat variables will change depending on the guard
+    force = 6.0
+    stamina_damage = 10
+    stamina_cost = 5.0
+
+    hit_type = DAMAGE_BLUNT
+    flags = FPRINT | TABLEPASS | USEDELAY
+    c_flags = EQUIPPED_WHILE_HELD
+    item_function_flags = USE_INTENT_SWITCH_TRIGGER | USE_SPECIALS_ON_ALL_INTENTS
+
+    var/guard
+
+    proc/change_guard(var/mob/user,var/intent)
```

Comment:
What happens if I have one of these in my hand and try to give it to someone else? Does this proc get called (might be via `dropped`, not sure when that gets called in that case, though)?

Reply:
oooh i haven't tested that yet.

ill look into it :D

Reply:
The interaction isn't 100% clean, but it does get reset to it's help intent stats on give, so it's not abusable